tests/lib/rules/array-callback-return.js

```diff
@@ -16,72 +16,103 @@ const ruleTester = new RuleTester();
 
 const allowImplicitOptions = [{ allowImplicit: true }];
 
+const checkForEachOptions = [{ checkForEach: true }];
+
+const allowImplicitCheckForEach = [{ allowImplicit: true, checkForEach: true }];
+
 ruleTester.run("array-callback-return", rule, {
     valid: [
 
-        // options: { allowImplicit: false }
-        "Array.from(x, function() { return true; })",
-        "Int32Array.from(x, function() { return true; })",
-        { code: "Array.from(x, function() { return true; })", options: [{ allowImplicit: false }] },
-        { code: "Int32Array.from(x, function() { return true; })", options: [{ allowImplicit: false }] },
```

Comment:
Looks like these two tests are removed?

Reply:
I reorganized these two without the explicit options declaration, they are on lines 50 and 51, is there any reason to explicitly declare `allowImplicit: false` for these cases?

Reply:
I reorganized these two without the explicit options declaration, they are on lines 50 and 51, is there any reason to explicitly declare `allowImplicit: false` for these cases?

Reply:
I think it's a nice regression test, personally.

Reply:
added options back